examples/gitlab-ha/README.md

```diff
@@ -35,17 +30,14 @@ First, copy `vars.mk.sample` to `vars.mk` and review/update the variables define
 
 * `DNS_ZONE_NAME` to the one associated with your AWS account
 * `REGION` to one that has some resources available to you
-* `REGISTRY_S3_BUCKET_NAME` to something unique
```

Comment:
Did this one get dropped somewhere along the line?

Reply:
mistake. will push fix